### PR TITLE
Reduce parachain template cognitive complexity

### PR DESCRIPTION
**Pull request by davxy**
Striving for simplicity, this PR removes some generics that are not really required in the template and thus reduces (at least a bit) some of the cognitive complexity required to understand the codebase by a new contributor

---

In particular here:
- some convenience type alias are defined
- remove generic `Runtime`. The parachain template is using only one runtime type

### CODE REVIEW

**Comment by davxy** [comment]
Added a type alias for `TFullClient` and `TFullBackend` in the `polkadot-parachain` crate as well.

This is opinionated, but I personally think that in general we should - to improve code readability - use more type aliases when we can.

File: parachain-template/node/src/service.rs [hunk 1]
```diff
@@ -52,56 +48,39 @@ impl sc_executor::NativeExecutionDispatch for TemplateRuntimeExecutor {
...
+
 /// Starts a `ServiceBuilder` for a full service.
 ///
 /// Use this macro if you don't actually need the full service, but just the builder in order to
 /// be able to perform chain operations.
 #[allow(clippy::type_complexity)]
-pub fn new_partial<RuntimeApi, Executor, BIQ>(
+pub fn new_partial<BIQ>(
 	config: &Configuration,
 	build_import_queue: BIQ,
```
Comment (bkchr) [inline parachain-template/node/src/service.rs:64]: This doesn't need to be configurable, whoever copied the code didn't know this. So, if you remove this, the code will get even less complicated :D

**Review by bkchr — APPROVED** [review]
Thank you! :) 

You can make it even simpler, otherwise it looks good.

**Comment by bkchr** [comment]
> Added a type alias for `TFullClient` and `TFullBackend` in the `polkadot-parachain` crate as well.
> 
> This is opinionated, but I **personally** think that in general we should - in order to improve code readability - use more type aliases when we can. Especially where there are a _bazilion_ of trait bounds for generics :-)

Yeah, we also have in other places similar types aliases. Not sure why we never used them here.

File: parachain-template/node/src/service.rs (hunk 1 above)
Comment (davxy) [reply parachain-template/node/src/service.rs:64]: Indeed. I'll do it before merge. Ty

**Review by altonen — APPROVED** [review]

**Pull request merged**